CMakeLists.txt

```diff
@@ -2201,6 +2202,7 @@ add_library(FpClassify STATIC EXCLUDE_FROM_ALL src/util/fpclassify.cpp)
 
 if (CMAKE_CXX_COMPILER_ID MATCHES "Clang" AND CMAKE_CXX_SIMULATE_ID MATCHES "MSVC")
   target_compile_options(FpClassify PRIVATE /fp:precise)
+  target_compile_options(mixxx-lib PRIVATE -Wno-microsoft-cast)
```

Comment:
What does "-Wno-microsoft-cast" do? Normally it would be better to fix the warnings.

Reply:
MSVC an GCC compiler allows it to cast an object pointer to a function pointer. This is not described in the C++ standard.
Clang warns about this non standard compliant cast, but executes it as the other compilers. This warning is inhibited hereby.

Reply:
Do we use that conversion?

Reply:
Yes, that's why we got the warnings: https://github.com/mixxxdj/mixxx/blob/c3b808cf90e124d25f324c784c8887c77f2bd8fd/src/encoder/encoderfdkaac.cpp#L134-L139

Reply:
Ah ok, in this case we have a function pointer. I think we can avoid the warning by a proper explicit cast to void*, or a reinterpret cast. 

At this point we are also not interested in the actual address. We want to see which of the pointers is NULL. 
So we may change the debugging to explicit report that.